### PR TITLE
OBSDOCS-1036/TRACING-4106: mention that tempo gateway only supports gRPC for ingesting traces

### PR DESCRIPTION
**Pull request by max-cx**
:warning: This PR is tied to a product release date, which is currently set to **June 5** (Wednesday).

- If the merge review gets delayed, you're welcome to ping `mleonov` on Slack.
- If you have any questions, ping `mleonov` on Slack.

Signed-off-by: Andreas Gerstmayr <agerstmayr@redhat.com>

Edit of https://github.com/openshift/openshift-docs/pull/74946

Version(s):

4.12, 4.13, 4.14, 4.15, 4.16

Issue:

https://issues.redhat.com/browse/OBSDOCS-1036
https://issues.redhat.com/browse/TRACING-4106

Link to docs preview:

https://75557--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.html#distr-tracing-tempo-config-multitenancy_distr-tracing-tempo-configuring

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/distr-tracing-tempo-config-multitenancy.adoc [hunk 1]
```diff
@@ -9,6 +9,11 @@
 Multitenancy with authentication and authorization is provided in the Tempo Gateway service.
 The authentication uses OpenShift OAuth and the Kubernetes `TokenReview` API. The authorization uses the Kubernetes `SubjectAccessReview` API.
 
+[NOTE]
+====
+The Tempo Gateway service supports ingestion of traces only via the OTLP protocol over the gRPC framework. The OTLP protocol is not supported over the HTTP protocol.
```
Comment (andreasgerstmayr) [inline modules/distr-tracing-tempo-config-multitenancy.adoc:14]: I think it sounds a bit weird. Maybe we should just refer it as `OTLP/gRPC` and `OTLP/HTTP`?
Here are the upstream docs: https://opentelemetry.io/docs/specs/otlp/

```suggestion
The Tempo Gateway service supports ingestion of traces only via OTLP/gRPC. OTLP/HTTP is not supported.
```

Comment (max-cx) [reply modules/distr-tracing-tempo-config-multitenancy.adoc:14]: @andreasgerstmayr, let's try it like this. I hope the peer reviewer won't have any issues with this.

**Review by IshwarKanse — APPROVED** [review]

**Comment by max-cx** [comment]
Closing this PR because I cherry-picked its commit into https://github.com/openshift/openshift-docs/pull/76340.

**Pull request merged**